src/stress/device/buffer_allocation.spec.ts

```diff
@@ -0,0 +1,27 @@
+export const description = `
+Stress tests for allocation of GPUBuffer objects through GPUDevice.
+`;
+
+import { makeTestGroup } from '../../common/framework/test_group.js';
+import { GPUTest } from '../../webgpu/gpu_test.js';
+
+export const g = makeTestGroup(GPUTest);
+
+g.test('concurrent_buffer_allocation')
+  .desc(`Tests allocation of many concurrent GPUBuffer objects.`)
+  .unimplemented();
+
+g.test('continuous_buffer_allocation_with_destroy')
+  .desc(
+    `Tests allocation and destruction of many GPUBuffer objects over time. Objects
+are sequentially created and destroyed over a very large number of iterations.`
+  )
+  .unimplemented();
```

Comment:
Some of these stress tests shouldn't in theory put much load on the system, unless there are bugs. For example in this issue:
https://bugs.chromium.org/p/dawn/issues/detail?id=914
Sunny is looking at memory allocation tests that _could_ be classified as stress tests, but the hope was that while they'll stress the code, they won't stress the system. (TBD whether that works out, I guess.)

So on the one hand, maybe some of what you have in this PR could be conformance tests (src/webgpu/).
But on the other hand, we should probably try to keep the conformance tests responsive (under the WPT timeouts). So it occurs to me it might be best to categorize all long-running tests as "stress" tests, instead of the split I had previously been imagining.

Reply:
Ruminating on this some more I do think @sunnyps's test should become a stress test, as it would be best if we could run it for longer periods.

See below, though, it might actually belong more in `stress/memory/churn.spec.ts` depending on what distinction we want between this test and that test. (Maybe we don't want any? We could just dedup them.)

Reply:
For the memory tests, I think de-duping probably makes sense. It sounds like @sunnyps's work will cover the churn cases that I was imagining, and probably more.

Regarding the general non-stressness of some tests, I wasn't really able to gauge how likely each plan would be to put a ton of load on the system, so my heuristic so far has been a pretty simplistic, "make number big."

From there my plan was to build the most stressful version of each test I can come up with, and if it turns out to not have a long running time in practice, move it to a regular conformance test.

And I guess finally, if there turn out to only be a handful of useful, very-long-running tests, maybe we just make them regular conformance tests with longer timeouts, or some mechanism to make them optional in CI, etc?

Does that sound reasonable?